static/src/javascripts/projects/commercial/modules/article-aside-adverts.spec.js

```diff
@@ -0,0 +1,124 @@
+// @flow
+/* eslint-disable no-new*/
+import qwery from 'qwery';
+import mediator from 'lib/mediator';
+import { noop } from 'lib/noop';
+import { articleAsideAdvertsInit } from 'commercial/modules/article-aside-adverts';
+import { commercialFeatures } from 'commercial/modules/commercial-features';
+import fastdom from 'lib/fastdom-promise';
+
+jest.mock('commercial/modules/commercial-features', () => ({
+    commercialFeatures: {
+        articleAsideAdverts: true,
+    },
+}));
+
+const mockMainColHeight = (height: number) => {
+    jest.spyOn(fastdom, 'read').mockReturnValue(Promise.resolve(height));
```

Comment:
I used a spy because I only want to mock `read`, I need `write` to still do its thing. Is that possible with a mock?